cmds/record.c

```diff
@@ -59,6 +63,20 @@ static int thread_ctl[2];
 static bool has_perf_event;
 static bool has_sched_event;
 
+static void check_shmem_freespace()
+{
+	struct statvfs stat;
+	long space;
+
+	if (statvfs(SHMEM_PATH, &stat) != 0) {
+		pr_err("Cannot access shared memory.\n");
+		return;
```

Comment:
You don't need to add '\n' in the `pr_err()`.  And 'return' is not necessary too since it calls `exit()` internally.